src/ImageSharp.Web.Providers.Azure/SeviceCollectionExtensions.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Six Labors.
+// Licensed under the Six Labors Split License.
+
+using Microsoft.Extensions.DependencyInjection;
+using SixLabors.ImageSharp.Web.Caching.Azure;
+using SixLabors.ImageSharp.Web.DependencyInjection;
+using SixLabors.ImageSharp.Web.Providers.Azure;
+
+namespace SixLabors.ImageSharp.Web;
+
+/// <summary>
+/// Extension methods for <see cref="IServiceCollection"/> to simplify middleware service registration.
+/// </summary>
+public static class SeviceCollectionExtensions
```

Comment:
I don't think we need these extensions. I'd rather we kept the API consistent across all provider types.

Reply:
Ok. Gonna remove that file again. But I think that there is a error on the website. 

![image](https://user-images.githubusercontent.com/873751/223030420-eb8d8b5d-ce7a-4e0e-8cfb-a29e34f8d78b.png)

This must be .SetCache<Adblob..>() or am I missing something?

Reply:
The `<` `>` tags were not HTML encoded. Have updated them now thanks.